File: wooden/proxy.js
const http = require('http');
const httpProxy = require('http-proxy');

const proxy = httpProxy.createProxyServer({});

function handleRequest(req, res) {
    // Proxy the request to the Google Sites page
    proxy.web(req, res, { target: 'https://sites.google.com/psdschools.org/wooden?usp=sharing', changeOrigin: true });
}

const server = http.createServer(handleRequest);

server.listen(3000, () => {
    console.log('Proxy server is running on http://localhost:3000');
});

File: wooden/proxy.py
from flask import Flask, request, Response
import requests

app = Flask(__name__)

@app.route('/')
def proxy():
    target_url = 'https://sites.google.com/psdschools.org/wooden?usp=sharing'
    response = requests.get(target_url)
    return Response(response.content, content_type=response.headers['Content-Type'])

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=3000)